runtime/vm/jvminit.c

```diff
@@ -3893,6 +3893,16 @@ processVMArgsFromFirstToLast(J9JavaVM * vm)
 		}
 	}
 
+	{
+		IDATA enableJVMRestorePortableeMode = FIND_AND_CONSUME_VMARG(EXACT_MATCH, VMOPT_XXENABLEJVMRESTOREPORTABLEMODE, NULL);
+		IDATA disableJVMRestorePortableeMode = FIND_AND_CONSUME_VMARG(EXACT_MATCH, VMOPT_XXDISABLEJVMRESTOREPORTABLEMODE, NULL);
+		if (enableJVMRestorePortableeMode >= disableJVMRestorePortableeMode) {
```

Comment:
This should be `>` right? Otherwise by default we're going to set `J9VM_CRIU_IS_PORTABLE_JVM_RESTORE_MODE` when no options are passed.